app/controllers/response_plans_controller.rb

```diff
@@ -6,39 +6,32 @@ class ResponsePlansController < ApplicationController
   before_action :authenticate_officer!
   before_action :authorize_admin, except: [:index, :show]
 
-  def new
```

Comment:
We can no longer create a new response plan without it being tied to a Person record.

Should clear this with Meredith.